server/chat.ts

```diff
@@ -1141,6 +1116,53 @@ export class CommandContext extends MessageContext {
 
 		return message;
 	}
+	canPM(targetUser: User) {
+		const user = this.user;
+
+		// for hotpatching, remove later.
+		Chat.updateSettings(user);
+		Chat.updateSettings(targetUser);
+
+		function checkBlocked(curUser: User, tarUser: User) {
```

Comment:
i do not like this nested function - this looks like it can be lifted to the top level (and then `tarUser` can be renamed to `targetUser` etc when its no longer shadowing). if this has to be inline, prefer lambda notation, `function` lifts names to the upper scope

Reply:
I went with inline, so it's easier to edit.